### DISCUSSION TITLE
<title>Thank you for giving me shark badge

### DISCUSSION THREAD

**Discussion by 521sai** [community/community#47671] [Other Feature Feedback, Questions, & Ideas]
#47671 
### Select Topic Area

Product Feedback

### Body

This badges help to recognize the good weightage git profile to shared in a resume

**Comment by Ntshangase** [comment]
Hi,

You can definitely get more badges, just check out [https://github.com/Schweinepriester/github-profile-achievements](url). 

If you found this helpful you know what to do, I'm trying get that galaxy brain badge🙇‍♂️
